Update deploy hooks page (#1689)
zeit/docs: pages/docs/v2/more/deploy-hooks.mdx

  description:
    'Deploy Hooks allow you to create URLs that accept HTTP POST requests to trigger deployments. Each URL correspond to a branch of a GitHub or GitLab repository linked to the project.',
  editUrl: 'pages/docs/v2/more/deploy-hooks.mdx',
  lastEdited: '2020-03-05T04:46:02.000Z'
}

Deploy Hooks allow you to create URLs that accept HTTP POST requests in order to trigger deployments and re-run the [Build Step](/docs/v2/build-step). These URLs are uniquely linked to your project, repository, and branch, so there is no need to use any authentication mechanism or provide any payload to the POST request.

This feature allows you to integrate ZEIT Now deployments with other systems. For example, you can set up:

- Automatic deployments by hooking into content changes in Headless CMSs.

> - Scheduled deployments by configuring cron jobs to trigger the Deploy Hook.

Reviewer: ```suggestion
- Scheduled deployments by configuring third-party cron job services to trigger the Deploy Hook.
```

- Forced deployments from the command line.

## Creating a Deploy Hook

To create a Deploy Hook for your project, make sure your project is linked to a Git repository using our [Git integrations](/docs/v2/git-integrations).

<Image
  src={`${process.env.ASSETS}/docs/common/github-from-existing-code.png`}
  width={1170/2}
  height={624/2}
  shadow
  borderRadius
/>
<Caption>Importing a project using our Git integrations.</Caption>

> Once your project is connected, go to its settings and you will see a section called "Deploy Hooks". Here, choose a name for your Deploy Hook and select the branch that will be deployed when the generated URL is requested.

Reviewer: ```suggestion
Once your project is connected, navigate to its "Settings" page and you will see a section called "Deploy Hooks". Here, choose a name for your Deploy Hook and select the branch that will be deployed when the generated URL is requested.
```

<Image
  src={`${process.env.ASSETS}/docs/advanced/deploy-hooks/deploy-hooks.png`}
  width={1462/2}
  height={448/2}
  shadow
  borderRadius
/>
<Caption>Creating a new Deploy Hook.</Caption>

<Note>
  We suggest you use a name that easily identifies the Deploy Hook so you will

>   be able to see when it triggers a deployment. We also suggest creating only

Reviewer: ```suggestion
  be able to understand when it triggers a deployment. We also suggest creating only
```

  already identifies your linked project.
</Note>

> After sending a POST request, you can see that it triggered a deploy on your project dashboard.

Reviewer: ```suggestion
After sending a POST request, you can see that it triggered a deployment on your project dashboard.
```

## Technical Details

> - Triggering a Deploy Hook will not rebuild an existing deployment. Instead, it will create a new deployment using the latest code from the specified branch.

Reviewer: ```suggestion
- Triggering a Deploy Hook will not rebuild an existing deployment. Instead, it will create a new deployment using the latest source code available on the specified branch.
```

- Currently, no build cache is used when triggering a Deploy Hook.

> - If you send multiple requests to deploy the same version of your project, subsequent deployments in queued jobs will be canceled to reduce build times.

Reviewer: ```suggestion
- If you send multiple requests to deploy the same version of your project, previous deployments for the same Deploy Hook will be canceled to reduce build times.
```
